ykman/cli/fido.py

```diff
@@ -46,11 +50,14 @@ def fido(ctx):
     """
     Manage FIDO applications.
     """
-    try:
-        ctx.obj['controller'] = Fido2Controller(ctx.obj['dev'].driver)
-    except Exception as e:
-        logger.debug('Failed to load Fido2Controller', exc_info=e)
-        ctx.fail('Failed to load FIDO 2 Application.')
+    if ctx.obj['dev'].is_fips:
+        ctx.obj['controller'] = FipsU2fController(ctx.obj['dev'].driver)
```

Comment:
Can this fail too? Maybe do a generic try-catch around the whole thing?

Reply:
Good catch, thanks!